src/main/java/com/laserfiche/repository/api/RepositoryApiClient.java

```diff
@@ -55,4 +55,6 @@ public interface RepositoryApiClient {
      * @return The Laserfiche Repository Template Definitions API client.
      */
     TemplateDefinitionsClient getTemplateDefinitionClient();
+
+    void setLoadTestHeader(String testHeader);
```

Comment:
This is for test only and it seems like we could hide it from the public.

Reply:
Like @jason-chan-lf said in another comments, we could add that method to `RepositoryApiClientImpl` instead of the public interface.

Reply:
ok removed it from the interface